example/ios/Podfile

```diff
@@ -4,8 +4,8 @@ source 'https://github.com/m0nac0/flutter-maplibre-podspecs.git'
 # Uncomment this line to define a global platform for your project
 platform :ios, '11.0'
 
-pod 'MapLibre'
-pod 'MapLibreAnnotationExtension'
+pod 'MapLibre', '5.12.2'
+pod 'MapLibreAnnotationExtension', '0.0.1-beta.2'
```

Comment:
We removed `MapLibreAnnotationExtension` in #277 and pinned it to `5.14.0-pre1`.

Reply:
I would suggest that remove `MapLibreAnnotationExtension` and add it back once we have a flutter api that uses which is not the case right now

Reply:
I guess #277 can make this PR obsolete. My only question would be, if developers still need to specify the dependency in the example/ios/Podfile at all?
If yes, we can change the that file and the Readme to the new version in #277

Reply:
@kuhnroyal Do you have some insight into whether developers still need to list the dependency in their podfile like we do in the example app, and if yes, is there some way we can tell Flutter to automatically add it to their Podfile?

Reply:
If `MapLibre` is available in cocaopods trunk (which it seems to be: https://github.com/CocoaPods/Specs/tree/master/Specs/b/0/0/MapLibre) and `MapLibreAnnotationExtension` is removed, then there is no need to specify anything manually.
If `MapLibreAnnotationExtension` is required again in the future, then it should also be published via cocoapods and then no manual listing is required either.

Reply:
Thank you, that sounds good. So I'll close this PR and open a new one to remove these lines.